users/models.py

```diff
@@ -1,95 +1,99 @@
 
+
 # 그룹 테이블
 class Group(accounts.models.BaseModel):
-    code = models.CharField(max_length=10, unique=True)     #그룹코드
+    code = models.CharField(max_length=10, unique=True)  # 그룹코드
+
 
 # 캐릭터 테이블
 class Character(accounts.models.BaseModel):
-    class CharacterType(models.IntegerChoices): #캐릭터 이미지
-        MOM = 0         #엄마
-        DAD = 1         #아빠
-        BROTHER1 = 2    #형제1
-        BROTHER2 = 3    #형제2
-        BROTHER3 = 4    #형제3
+    class CharacterType(models.IntegerChoices):  # 캐릭터 이미지
+        MOM = 0  # 엄마
+        DAD = 1  # 아빠
+        BROTHER1 = 2  # 형제1
+        BROTHER2 = 3  # 형제2
+        BROTHER3 = 4  # 형제3
 
-    id = models.PositiveIntegerField(primary_key=True)          #id
-    type = models.IntegerField(choices = CharacterType.choices) #캐릭터 이미지
+    id = models.PositiveIntegerField(primary_key=True)  # id
+    type = models.IntegerField(choices=CharacterType.choices)  # 캐릭터 이미지
 
 
 # 유저 정보 테이블
 class Info(accounts.models.BaseModel):
-    class ActivityType(models.IntegerChoices):      #활동수준
-        NOCHOICES = -1  #선택을 안 했을 때(비 당뇨인의 경우)
-        DEEPLOW = 0     #활동이 적거나 운동을 안하는 경우
-        LOW = 1         #가벼운 활동 및 운동
-        MIDDLE = 2      #보통의 활동 및 운동
-        HIGH = 3        #적극적인 활동 및 운동
-        DEEPHIGH = 4    #매우 적극적인 활동 및 운동
-
-    class GenderType(models.TextChoices):           #성별
-            FEMALE = 'f'
-            MALE = 'm'
-
-    # id                                                                                            #로그인테이블 ID
-    name = models.CharField(max_length=20)                                                          #이름
-    character_id = models.ForeignKey(Character, on_delete=models.SET_NULL, null=True)               #캐릭터 ID
-    height = models.FloatField(default=None, null=True)                                             #키
-    weight = models.FloatField(default=None, null=True)                                             #몸무게
-    gender = models.CharField(max_length=1, choices = GenderType.choices)                           #성별
-    is_diabetes = models.BooleanField(default=False)                                                #당뇨환자여부
-    activity = models.IntegerField(default=ActivityType.NOCHOICES, choices = ActivityType.choices)  #활동수준
-    # account_user_group_id                                                                         #그룹코드 ID
+    class ActivityType(models.IntegerChoices):  # 활동수준
+        NOCHOICES = -1  # 선택을 안 했을 때(비 당뇨인의 경우)
+        DEEPLOW = 0  # 활동이 적거나 운동을 안하는 경우
+        LOW = 1  # 가벼운 활동 및 운동
+        MIDDLE = 2  # 보통의 활동 및 운동
+        HIGH = 3  # 적극적인 활동 및 운동
+        DEEPHIGH = 4  # 매우 적극적인 활동 및 운동
+
+    class GenderType(models.TextChoices):  # 성별
+        FEMALE = 'f'
+        MALE = 'm'
+
+    id = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)  # 로그인테이블 ID
+    name = models.CharField(max_length=20)  # 이름
+    character_id = models.ForeignKey(Character, on_delete=models.SET_NULL, null=True)  # 캐릭터 ID
```

Comment:
```suggestion
    character = models.ForeignKey(Character, on_delete=models.SET_NULL, null=True)  # 캐릭터 ID
```
외래키는 장고에서 뒤에 id를 자동으로 붙여줘서, character로 해도 될 것 같습니다!
https://naver.me/xbnozokp